hapi/hedera-protobufs/services/custom_fees.proto

```diff
@@ -173,3 +173,15 @@ message AssessedCustomFee {
    */
   repeated AccountID effective_payer_account_id = 4;
 }
+
+message ConsensusCustomFee {
+  /**
+  * Fixed fee to be charged
+  */
+  FixedFee fixed_fee = 1;
+
+  /**
+  * The account to receive the custom fee
+  */
+  AccountID fee_collector_account_id = 2;
+}
```

Comment:
1. Why are we defining a whole new custom fee type here?  Would it not make more sense to use the existing custom fee structure?
1. Specification comments should specify behavior of the field and follow correct formatting
1. Messages must have specification comments that describe the use of that message and interactions with other systems (e.g. Block Stream), if any.
```suggestion

/**
 * A custom fee definition for a consensus topic.
 *
 * This fee definition is specific to an Hedera Consensus Service (HCS) topic
 * and SHOULD NOT be used in any other context.<br/>
 * All fields for this message are REQUIRED.<br/>
 * Only "fixed" fee definitions are supported because there is no basis for
 * a fractional fee on a consensus submit transaction.
 */
message ConsensusCustomFee {
    /**
     * A fixed custom fee.
     * <p>
     * The amount of HBAR or other token described by this `FixedFee` SHALL
     * be charged to the transction payer for each message submitted to a
     * topic that assigns this consensus custom fee.
     */
    FixedFee fixed_fee = 1;

    /**
     * A collection account identifier.
     * <p>
     * All amounts collected for this consensus custom fee SHALL be transferred
     * to the account identified by this field.
     */
    AccountID fee_collector_account_id = 2;
}

```
Note: I am guessing a bit as to behavior in this suggestion; please correct any errors above.

Reply:
1. This is under discussion with @Neurone . 
2 + 3. Fixed